### PR TITLE
Fully revised and fool-proof docker guide

### PR DESCRIPTION
**Pull request by willysunny**
With an explanation for every single command and what it does.

### CODE REVIEW

File: docs/guides/Docker Guide.md [hunk 1]
```diff
@@ -1,66 +1,106 @@
...
+  - Windows 8 and earlier
+- For Mac
+  - Any version newer than 10.8 “Mountain Lion”
+  - If your mac is newer than 10.10.3 Yosemite, keep on reading this guide instead.
+
+## Prerequisites
+- [Docker](https://store.docker.com/search?type=edition&offering=community) or [Docker Toolbox](https://www.docker.com/products/docker-toolbox).
 - [Create Discord Bot application](http://nadekobot.readthedocs.io/en/latest/JSON%20Explanations/#creating-discord-bot-application) and [Invite the bot to your server](http://nadekobot.readthedocs.io/en/latest/JSON%20Explanations/#inviting-your-bot-to-your-server). 
+- Have your [credentials.json](http://nadekobot.readthedocs.io/en/latest/JSON%20Explanations/#setting-up-your-credentials) in your home folder.
+  - Home folder for linux/mac: **~**
```
Comment (Macley-Kun) [inline docs/guides/Docker Guide.md]: For the part about home folders on linux and mac you can change it to smth like this:
For linux you can do: cd $HOME to go to your home dir
Fox mac: https://www.cnet.com/how-to/how-to-find-your-macs-home-folder-and-add-it-to-finder/ (you do command Shif H)

tldr:
Home folder for Linux: cd $HOME
Home folder for MacOS: ⌘ + Shift + H

File: docs/guides/Docker Guide.md [hunk 2]
```diff
@@ -1,66 +1,106 @@
...
-#### Prerequisites
-- [Docker](https://docs.docker.com/engine/installation/)
+## Before you start ...
+... If your PC falls under any of the following cases, please grab [Docker Toolbox](https://www.docker.com/products/docker-toolbox) instead.
+- For Windows
+  - Any Windows version without Hyper-V Support
+  - Windows 10 Home Edition
+  - Windows 8 and earlier
+- For Mac
+  - Any version newer than 10.8 “Mountain Lion”
```
Comment (Macley-Kun) [inline docs/guides/Docker Guide.md]: Maybe reword this in smth like:
Any version between 10.8 “Mountain Lion” and 10.10.2 Yosemite.

Because your summing up the versions that don't support. It might be a little confusing if you say that *from 10.10.3* your good again.

**Comment by MaybeGoogle** [comment]
More fool-proof: download and run the scripts, depending on OS.
[OSX](https://raw.githubusercontent.com/MaybeGoogle/NadekoScripts/master/Nadecker_Mac.sh)
[Windows](https://raw.githubusercontent.com/MaybeGoogle/NadekoScripts/master/Nadecker_Windows.ps1)

**Comment by Kwoth** [comment]
I'll test it soon:tm: and then I'll merge if everything works.

**Comment by Kwoth** [comment]
thanks
